tests/unit/serve/runtimes/gateway/grpc/test_grpc_gateway_runtime.py

```diff
@@ -158,6 +158,20 @@ async def task_wrapper():
         return asyncio.create_task(task_wrapper())
 
 
+async def _test(streamer):
+    responses = []
+    req = request_generator('/', DocumentArray([Document(text='client0-Request')]))
+    async for resp in streamer.stream(request_iterator=req):
+        responses.append(resp)
+    for req in request_generator(
+        '/',
+        DocumentArray([Document(text='client0-Request')]),
+    ):
+        unary_response = await streamer.process_single_data(request=req)
+        responses.append(unary_response)
+    return responses
```

Comment:
isn't it better to add option stream as a parameter to this function and as a parameter to the test ? that way we'll be testing each feature on its own

Reply:
It can be done. The tests will be parametrized and will run both anyways. If it's about isolating tests then I'll refactor.